packages/blade/src/components/Dropdown/useDropdown.ts

```diff
@@ -216,6 +224,7 @@ const useDropdown = (): UseDropdownReturnValue => {
     setChangeCallbackTriggerer,
     isControlled,
     setControlledValueIndices,
+    filteredValues,
     ...rest
```

Comment:
nit: Why `rest`? There are only a few other values, lets destruct them all here na?

Reply:
oh yup we'll do

Reply:
oh I remembered. So we re-export all dropdown context values from useDropdown + some additional functions.

If we destructure here, it becomes 2 more places to go on adding every value we add to dropdown context. Its already a bit pain because we have to add it to to value in provider, type, initial type, dependency array of Dropdown's useMemo.

I think its better to leave it to rest cause there are around 14 properties that we'll have destructure here and then again add them to return value of this function